templates/hotgraphic.hbs

```diff
@@ -23,7 +23,7 @@
 
         <button class="btn-icon hotgraphic__pin item-{{@index}}{{#if _graphic._classes}} {{_graphic._classes}}{{/if}}{{#if _pin.src}} has-pin-image{{/if}}{{#if _isVisited}} is-visited{{/if}} js-hotgraphic-item-click" data-index="{{@index}}" style="top: {{{_top}}}%; left: {{{_left}}}%;">
 
-          <span class="aria-label">{{#if ../_useNumberedPins}}{{math @index "+" 1}} {{/if}}{{#if _pin.alt}}{{{compile _pin.alt}}}{{else}}{{{compile title}}}{{/if}}.</span>
+          <span class="aria-label">{{#if ../_useNumberedPins}}{{math @index "+" 1}} {{/if}}{{#if _pin.alt}}{{{compile _pin.alt}}}{{else}}{{{compile title}}}{{/if}}. Item {{math @index "+" 1}} of {{../_items.length}}</span>
```

Comment:
The index between the forward and back arrows in the function updatePageCount should also be in plain English and translatable, rather than the format x / y

Reply:
@StuartNicholls With the next and back arrows now reading out the item number and item I've suggested we hide the current x / y from the screen reader completely, do you think that would be better or prefer to have that spoken out in full English as well?

Reply:
I was just referring to general legibility rather than screen readers.

I can't actually test the screen reader functionality at the moment but I'm not seeing that screen reader text on those buttons other than 'previous' and 'next' - just on the pins. But, if thats the case I'd imagine the page index wasn't necessary for screen readers.

Reply:
Next / previous icons also need aria-hidden in the version I'm looking at